sotodlib/site_pipeline/update_obsdb.py

```diff
@@ -235,7 +235,12 @@ def main(config: str,
             if stream_ids is not None:
                 bookcartobsdb.add_obs_columns(["wafer_count int"])
                 very_clean["wafer_count"] = len(stream_ids)
-
+                bookcartobsdb.add_obs_columns(["ufms str"])
+                very_clean["ufms"] = (",".join(stream_ids)).replace("ufm_","")
+            wafer_slots = index.pop("wafer_slots")
+            if wafer_slots is not None:
+                bookcartobsdb.add_obs_columns(["wafer_slots str"])
+                very_clean["wafer_slots"] = (",".join([sl["wafer_slot"] for sl in wafer_slots]))
```

Comment:
Thanks for this -- but how do odd computed things like this play with update_obsdbcols?

Would it make sense to have a function that does "collect all obsdb data for book _X_", and then call that function from both update_obsdb.py and update_obsdbcols.py?

I think you should also consider just putting all of this capability into update_obsdb.py.  There's a lot of duplicate logic + boilerplate already, so your new code won't add too many lines; it could be nice to have all the obsdb building / rebuilding in one place.